src/widgets/table.rs

```diff
@@ -60,6 +60,8 @@ where
     /// Width of each column (if the total width is greater than the widget width some columns may
     /// not be displayed)
     widths: &'a [u16],
+    /// Percentage width of each column, allows table to expand horizontally to fill space
+    percentage_widths: &'a [u16],
```

Comment:
Have you given thoughts about using an `Enum` that would allow `Fixed` and `Percent` width in `widths` instead ?

Reply:
I did think about it, it would be cool to have but I wasn't sure about introducing a breaking change here.

Reply:
I already have breaking changes planned, so I'm fine with this one =)